src/main/java/org/gradle/profiler/studio/LauncherConfigurationParser.java

```diff
@@ -46,9 +46,12 @@ public LaunchConfiguration calculate(Path studioInstallDir, String pluginPort) {
         Path studioPlugin = GradleInstrumentation.unpackPlugin("studio-plugin").toPath();
         Path studioPluginsDir = newPluginTempDir();
 
-        systemProperties.put("idea.plugins.path", studioPluginsDir.toAbsolutePath().toString());
-        systemProperties.put("idea.log.path", studioPluginsDir.toAbsolutePath().toString());
-        return new LaunchConfiguration(javaCommand, classPath, systemProperties, mainClass, agentJar, supportJar, Arrays.asList(asmJar, protocolJar), Arrays.asList(studioPlugin, protocolJar), studioPluginsDir);
+        String studioPluginsDirPath = studioPluginsDir.toAbsolutePath().toString();
+        List<Path> sharedJars = Arrays.asList(asmJar, protocolJar);
+        List<Path> studioPluginJars = Arrays.asList(studioPlugin, protocolJar);
+        systemProperties.put("idea.plugins.path", studioPluginsDirPath);
+        systemProperties.put("idea.log.path", studioPluginsDirPath);
```

Comment:
Readability: is there a reason not to swap line 50-51 with line 52-53?

Reply:
fixed